Leauge Auto Accept/UI.cs

```diff
@@ -49,11 +49,12 @@ public static bool showCursor
                 }
                 showCursorValue = value;
             }
-        }
+        }   
 
         public static void initializingWindow()
         {
             Print.canMovePos = false;
+            Console.Clear();

```

Comment:
If Console.Clear() is excluded here the "Console too small please resize" message is still on the screen with the "Initializing..." message